### ISSUE TITLE
Interest in packaging jchempaint as Flatpak for Linux?

### ISSUE THREAD

**Issue by kevinsmia1939** [JChemPaint/jchempaint#195]
Hi,
Are you interest in putting jchempaint on Flathub when packaging as Flatpak for Linux?

Flatpak is Linux software manager and deployment of Linux.
https://flatpak.org/

I could try to do it. What do you think?

**Comment by Mailaender** [comment]
Yes, I created https://github.com/flathub/flathub/pull/6273 where you @egonw are of course also invited.

**Comment by egonw** [comment]
It is online now as https://flathub.org/apps/io.github.jchempaint.JChemPaint
